extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -221,40 +228,156 @@ class AmpAccordion extends AMP.BaseElement {
     const content = sectionComponents[1];
     const contentId = content.getAttribute('id');
     const isSectionClosedAfterClick = section.hasAttribute('expanded');
-    this.mutateElement(() => {
-      let toExpand;
-      if (opt_forceExpand === true) {
-        toExpand = true;
-      } else if (opt_forceExpand === false) {
-        toExpand = false;
-      } else {
-        toExpand = !section.hasAttribute('expanded');
-      }
+    let toExpand;
+    if (opt_forceExpand === true) {
```

Comment:
cleanup nit: this tri-state is a little confusing. I would refactor to avoid repeating `if (true) { variable = true } else if (false) { variable = false }`

```js
const toExpand = (opt_forceExpand === undefined ? !section.hasAttribute(expanded) : opt_forceExpand)